ivoatexDoc.tex

```diff
@@ -1045,22 +1068,51 @@ \subsection{Generated Content}
 to produce documentation for the schema type \xmlel{MyType} within
 the XML schema file \texttt{SchemaSource.xsd}.
 
-\ivoatex~will never execute \texttt{update\_generated.py} as part of a
-dependency chain; it is intended that \texttt{make generate} must always
-been manually triggered.  On the one hand, this is because its
-dependencies cannot be generically modelled, given that arbitrary
-commands can be executed.  Document authors are also discouraged from
-providing such dependency information -- it is fairly common that
-content generation depends on the availability of external resources
-(e.g., databases or network services), and a document build should not
-fail just because these are unavailable.
+\paragraph{vocterms}
 
-We mention in passing that generated content puts potentially executable
-material into documents, which is of course an attack vector for
-malicious software.  However, calling \texttt{make generate} is no
-additional security risk, as whoever can change the document can
-probably change the makefile, too, and the makefile can already contain
-arbitrary commands that will be executed on the calling user's behalf.
+The \texttt{vocterms} builtin formats the identifiers for an IVOA
+vocabulary \citep{2021ivoa.spec.0525D} into a comma-separated list.  It
+takes one argument, the vocabulary name (or, for legacy vocabularies
+that have path-like names, the relative path; in practice: whatever is
+behind \texttt{rdf/} in the vocabulary URI).  For instance,
+
```

Comment:
Since I'm not a tex guru and didn't look at update_generated, I'm assuming some things about how it works and the intended process for authoring documents... 

For this new vocterms builtin, you should probably include the sample tex fragment here (for clarity), indicate explicitly that it is `vocterms.tex` that gets modified, not the main doc, and maybe clarify that the included tex fragment would normally be committed and added to version control. I might also rename `vocterms.tex` to `vocab-eample.tex` so it's clear that the filename doesn't matter and that the built in and the vocabulary are both specified in the comments in the tex fragment; that also makes it more clear that it's simple to have multiple of these included in a document because the filename doesn't actually matter.

If the above makes sense, then maybe `taptable` (which is changed to this "include tex fragment" style) should also be tweaked a bit to clarify in the same way, or the clarifications put further up as common advice.